README.md

```diff
@@ -178,6 +182,33 @@ To run just one python version:
 tox -e py36
 ```
 
+### Integration Tests
+
+The integration tests for ssh mailing list creation and LDAP require additional
+information to be run, which should not be published.
+By default, these tests are *skipped*, as you can see in the tox summary.
+
+#### SSH
```

Comment:
What does SSH have to do with this? What kind of SSH connection is required here?

Reply:
It's just a bit tangential. Since I updated the integration tests for ldap and needed to add that to the Readme, I included the ssh integration tests here as well while I was at it. No functionality changes.